pkg/cmd/admin/policy/who_can.go

```diff
@@ -69,12 +72,23 @@ func (o *whoCanOptions) run() error {
 	resourceAccessReview.Resource = o.resource
 	resourceAccessReview.Verb = o.verb
 
-	resourceAccessReviewResponse, err := o.client.ResourceAccessReviews(o.bindingNamespace).Create(resourceAccessReview)
+	var reviewInterface client.ResourceAccessReviewInterface
+	if o.allNamespaces {
+		reviewInterface = o.client.ClusterResourceAccessReviews()
+	} else {
+		reviewInterface = o.client.ResourceAccessReviews(o.bindingNamespace)
+	}
+
+	resourceAccessReviewResponse, err := reviewInterface.Create(resourceAccessReview)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("Namespace: %s\n", resourceAccessReviewResponse.Namespace)
+	if o.allNamespaces && resourceAccessReviewResponse.Namespace == kapi.NamespaceAll {
```

Comment:
Don't we want to raise error when o.allNamespaces = true and  resourceAccessReviewResponse.Namespace != kapi.NamespaceAll ?